database/mssql/scripts/dbscripts/PSP_PIMS_S10_00/Build/82_DML_LEASE_TESTING.sql

```diff
@@ -0,0 +1,55 @@
+VALUES
+  (N'45 - 904 Hollywood Crescent', N'Living in a van', N'Down by the River',                N'Hollywood North', N'M4T F0L', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'PO Box 2',                    N'Stealth Camping', N'Walmart Parking Lot',              N'South Podunk',    N'H1I B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'123 Main Street',             N'PO Box 123',      N'Next to the Dairy Queen',          N'East Podunk',     N'I4M B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data'),
+  (N'456 Souris Street',           N'PO Box 250',      N'A Hoot and a holler from the A&W', N'North Podunk',    N'IH8 B0B', 1, 1, N'PHYSADDR', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data', CURRENT_TIMESTAMP, N'Seed Data', N'Seed Data');
+
+-- Insert organiations
+INSERT INTO PIMS_ORGANIZATION (ORGANIZATION_IDENTIFIER, ORGANIZATION_NAME, ORGANIZATION_TYPE_CODE, ORG_IDENTIFIER_TYPE_CODE, ADDRESS_ID, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
+VALUES
+  (N'LILMOUSE', N'French Mouse Property Management',         N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Souris%'),    CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'DQ4EVA',   N'Dairy Queen Forever! Property Management', N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '123 Main%'),   CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'DAKATZ',   N'Pussycat Property Management',             N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE 'PO Box%'),     CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data'),
+  (N'BOB123',   N'Bob''s Property Management',               N'REALTOR', N'OTHINCORPNO', (SELECT ADDRESS_ID FROM PIMS_ADDRESS WHERE STREET_ADDRESS_1 LIKE '%Hollywood%'), CURRENT_TIMESTAMP, N'Lease Seed Data', N'Leasse Seed Data', CURRENT_TIMESTAMP, N'Leasse Seed Data', N'Leasse Seed Data');
+
+-- Insert people
+INSERT INTO PIMS_PERSON (ADDRESS_ID, SURNAME, FIRST_NAME, MIDDLE_NAMES, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
```

Comment:
this is also missing is_disabled.

Reply:
IS_DISABLED defaults to FALSE so I didn't feel the need to add to the SQL.  If you need it, let me know.